test/unit/web_framework/test_short_term_storage.py

```diff
@@ -7,7 +7,7 @@
     MessageException,
```

Comment:
If we agreed to move `short_term_storage` to the `app` package for the time being, then also this file should be moved to `test/unit/app/`

Reply:
Thanks! I will move it for now to test/unit/app/